projects/demo/src/modules/charts/line-chart/examples/6/index.less

```diff
@@ -0,0 +1,15 @@
+.axes {
+    height: 12.5rem;
+    width: 25rem;
+    color: #bc71c9;
+}
+
+.icon {
+    background: white;
```

Comment:
bad for dark mode

<img width="490" alt="dark-mode" src="https://user-images.githubusercontent.com/35179038/162927326-335aa3ef-ea1b-494e-bc84-c1ebfd2680c2.png">

___

Is `--tui-base-01` an appropriate alternative ?

Reply:
thanks! fixed